connectors/sources/jira.py

```diff
@@ -93,59 +92,75 @@ def get_default_configuration(cls):
         """
         return {
             "is_cloud": {
-                "value": True,
+                "display": "toggle",
                 "label": "True if Jira Cloud, False if Jira Server",
+                "order": 1,
                 "type": "bool",
+                "value": True,
             },
             "username": {
-                "value": "admin",
                 "label": "Jira Server username",
+                "order": 2,
                 "type": "str",
+                "value": "admin",
             },
             "password": {
-                "value": "changeme",
                 "label": "Jira Server password",
+                "sensitive": True,
+                "order": 3,
                 "type": "str",
+                "value": "changeme",
             },
             "service_account_id": {
-                "value": "me@example.com",
                 "label": "Jira Cloud service account id",
+                "order": 4,
                 "type": "str",
+                "value": "me@example.com",
             },
             "api_token": {
-                "value": "abc#123",
                 "label": "Jira Cloud API token",
+                "order": 5,
```

Comment:
Shouldn't we keep "sensitive": True, for api_token since api_tokens are directly used as passwords by the connector in case of Cloud deployment ?